locales/core.js

```diff
@@ -1,3 +1,9 @@
+var i18n = i18n;
```

Comment:
I don’t quite get that. Is this needed for Nashorn/Java?
For the browser and NodeJS it works without it.